components.json

```diff
@@ -0,0 +1,21 @@
+{
```

Comment:
I installed Shadcn as I've had a good experience using this library. It's quite user friendly, extensible, and looks nice.